aiosmtpd/tests/conftest.py

```diff
@@ -96,6 +132,14 @@ def cache_fqdn(session_mocker):
 # region #### Common Fixtures #########################################################
 
 
+@pytest.fixture(autouse=True)
+def log_case(request: pytest.FixtureRequest):
+    node_id = request.node.nodeid
+    log.debug("Entering %s", node_id)
+    yield
+    log.debug("Exiting %s", node_id)
```

Comment:
This makes test logs easier to read since we now see which test case triggered which sequence of ops.